routes/index.js

```diff
@@ -4,10 +4,13 @@ const router = express.Router();
 
 const { signin, signout } = require("../controllers/authController");
 const { getLayout } = require("../controllers/drumController");
+const { uploadSound } = require("../controllers/uploadController");
 const { requireAuth } = require("../middleware/auth");
 
 router.post("/signin", signin);
 router.post("/signout", signout);
 router.get("/drumlayout", requireAuth, getLayout);
+router.get("/uploadsound", uploadSound); //displays form to upload the sound
```

Comment:
- [ ] Can you explain what the GET endpoint is for? I'm not sure I understand what that would be used for.

Reply:
Since I was adding images for the snare, hi-hat, and bassdrum sounds, I was thinking this endpoint was for the programmers to upload the default sounds for these. I reviewed the outline of the api and see now that the intent is an endpoint for the users to create and upload a custom sound. 

I removed the get functionality, but this is my first experience with prs. I tried to create a new pr with the changes and it won't let me maybe because it conflicts with this pr? How do I proceed with a pr with my changes that remove the get functionality?

Thank you for your patience as I learn the ropes!!
Tisha

Reply:
I think this is ready to be looked over for approval.